lib/arjdbc/jdbc/adapter.rb

```diff
@@ -289,12 +289,14 @@ def active?
 
       # @override
       def reconnect!
+        super
```

Comment:
thanks, I am starting to recall this worked differently for some reason/purpose.
anyway its probably worth a new investigation how/when it changed. 
... although seems less important by now :) thanks again!